.bazelrc

```diff
@@ -25,7 +25,7 @@ build --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_java8
 build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
 
 # Import default javacopts
-import tools/javacopts.bazelrc
+import %workspace%/tools/javacopts.bazelrc
```

Comment:
Should the user import at the bottom be rooted as well?

Reply:
Nice catch. I think yes.